TestsFoundation/TestsFoundation/CoreUIElements/K5/K5Schedule.swift

```diff
@@ -20,4 +20,20 @@ import XCTest
 
 public enum K5Schedule: ElementWrapper {
     case today
+
+    public static var previousWeekButton: Element {
+        app.find(label: "Previous Week", type: XCUIElement.ElementType.button)
```

Comment:
In such cases the type of the `type` parameter is defined in the `find()` method signature so we can omit `XCUIElement.ElementType` and only write `.button`.